.travis/run.sh

```diff
@@ -4,10 +4,10 @@ set -x
 
 if [[ $TOXENV == py* ]]; then
     # Run unit tests
-    tox -- -m unit
+    tox --develop -- -m unit
     # Run integration tests
-    tox -- -m integration -n 4 --duration=5
+    tox --develop -- -m integration -n 4 --duration=5
```

Comment:
Why the change to use `--develop` here? I don't have a particular problem with it, just don't understand why it's needed (and whether that implies there's an implication of one of the other changes that I've missed).

Reply:
It's just faster, and we don't need to do a normal installation anymore (since we don't run unvendored tests).

Reply:
OK, cool, so not specifically related to the Windows changes. That's fine (and faster is always good ;-)).